README.md

```diff
@@ -1,50 +1,129 @@
+
+To deploy filebeat, elasticsearh and kibana it uses helm charts by elastic ([https://github.com/elastic/helm-charts](https://github.com/elastic/helm-charts)).
+
+## Monitor and Alerts
+
+Spacecraft deploys spacemesh-watch ([https://github.com/spacemeshos/spacemesh-watch](https://github.com/spacemeshos/spacemesh-watch)) service which uses GRPC API of the managed miners to monitor them. And when some issue is detected it send alerts in slack. 
+
+## Web Services
+
+Web services for a network is basically combination three things: dashboard, explorer and spacemesh public API.
+
+The dashboard and explorer frontend are already deployed in GCP storage and served as static website as load balancer. The dashboard and explorer backends needs to deployed separately for every network. The dashboard and explorer frontends looks for list of deployed networks in [discover.spacemesh.io/networks.json](http://discover.spacemesh.io/networks.json) file and allows the user to select a network via a dropdown.
+
+In spacecraft, when web services are deployed it updates the networks.json file with the information about the new network. The [discover.spacemesh.io/networks.json](http://discover.spacemesh.io/networks.json) file is hosted in GCP storage and served as static website using GCP load balancer.
+
+Currently web services aren't automatically deployed when a network is deployed instead you need to run the `deployWS` sub-command to deploy the web services. 
+
+For deploying web services, spacecraft uses the ws-helm-charts ([https://github.com/spacemeshos/ws-helm-charts](https://github.com/spacemeshos/ws-helm-charts)). 
+
+It deploys the dashboard backend and explorer backend as multiple replicas for higher availability. It also deploys explorer collector as multiple replicas for higher availability. The spacemesh public API is also deployed as multiple replicas. 
+
+Here is high level architecture diagram of the dashboard backend components and their interactions:
+
+![dash_arch_chart.png](docs/dash_arch_chart.png)
+
+Here is high level architecture diagram of the explorer backend components and their interactions:
+
+![explorer_arch_chart.png](docs/explorer_arch_chart.png)
+
+Cloudflare is used as SSL proxy and for domain records for spacemesh public JSON API, explorer backend API and dashboard backend API. Whereas for spacemesh public GRPC API, k8s Ingress  is used as SSL proxy and cloudflare is used only for DNS record. The reason we do this for GRPC API is because cloudflare is capable to proxy GRPC connections currently. Due to this we store SSL cert and key in secrets.
+
+## Metrics
+
+If the `--metrics` flag is set during network deployment then prometheus and grafana is deployed to collect metrics. Prometheus and grafana is deployed using helm charts. These are deployed in `metrics` namespace. 
+
+Here is the architecture of the deployment:
+
+![metrics.png](docs/metrics.png)
```

Comment:
These components are not described in `deployment_arch.png`. It should be added if possible. 
Also, are you sure that all of them are implemented?

Reply:
Yes they all are implemented. It's deployed in a separate namespace therefore I think it's fine to show it as a seperate architecture diagram.